tests/torch/test_brevitas_qat.py

```diff
@@ -606,3 +611,20 @@ def test_brevitas_power_of_two(
 
     check_array_equal(y_pred_sim_round, y_pred_clear_round)
     check_array_equal(y_pred_clear_round, y_pred_clear_no_round)
+
+
+def test_brevitas_channel_wise():
+    """Make sure that we can compile brevitas channel-wise quantization"""
+    model = QuantLeNet()
+    model.eval()
+
+    with torch.no_grad():
+        batch_size = 3
+        image_size = 1, 32, 32
+        images = torch.rand((batch_size, *image_size))
+        out = model(images).detach().numpy()
+        quantized_module = compile_brevitas_qat_model(model, images, rounding_threshold_bits=6)
+        out_qm = quantized_module(images.detach().numpy())
+        mse = ((out - out_qm) ** 2).mean()
+        # Arbitrary threshold to check that the predictions are relatively similar
+        assert mse < 1e-4
```

Comment:
probably better to use our tools like `check_float_array_equal`, which can provide rtol or atol, or `check_r2_score` or similar 

if not, then probably better to make this mse check a fixture so that we can re-use it elsewhere and not forget about this arbitrary value here

Reply:
yeah true `check_float_array_equal` would probably be better indeed. I'll change that.